benefits/locale/README.md

```diff
@@ -0,0 +1,36 @@
+# Copy process
+
+The `LC_MESSAGES` folders in this directory contain the `django.po` message files for English and Spanish translation strings for the Benefits application.
+
+Translation strings include all application copy, including:
+
+- All application copy
+- Image and illustration alt tags
+- Page titles used in the browser tab
+- In-line link URLs
+
+## BENEFITS STRING SPREADSHEET NAME TBD
+
+The human-readable version of the English and Spanish translation strings for the application are delivered to Design and Engineering by Product, and live at this link: [Benefits String Spreadsheet NAME TBD](https://docs.google.com/spreadsheets/d/13YZLv7wf8dAk_HnEP-KLEUurTggx8QOx9HovEW4_UjI/edit#gid=0).
+
+## Copy process responsibilities
+
+### Product
+
+- Engaging with product stakeholders to get the information necessary for copy writing.
+- Engaging with copy writers to get the English language copy drafted, proofed and ready for design.
+- Engaging with client editorial/communications team to ensure English language and Spanish language copy are edited according to client style guides.
+- Engaging all necessary stakeholders to get English language copy approved and ready for design.
+- Compiling copy in [Benefits String Spreadsheet NAME TBD](https://docs.google.com/spreadsheets/d/13YZLv7wf8dAk_HnEP-KLEUurTggx8QOx9HovEW4_UjI/edit#gid=0), ready to be used by Design, so Design can sync the spreadsheet to Figma.
+- Engaging with the translation agency, [iBabbleOn](https://ibabbleon.com/), to get Spanish translations ready for Engineering.
+- Ensure English and Spanish copy is ready for Engineering.
+
+### Design
+
+- Syncing copy from [Benefits String Spreadsheet](https://docs.google.com/spreadsheets/d/13YZLv7wf8dAk_HnEP-KLEUurTggx8QOx9HovEW4_UjI/edit#gid=0) into Figma.
+- Ensuring the string is in the appropriate column (e.g. `Subtitle`, `ButtonLabel`)
+
+### Engineering
```

Comment:
Need to add a reminder to run `bin/makemessages.sh`: https://github.com/cal-itp/benefits/pull/1436#discussion_r1242647512

Reply:
Now that #1571 is done, engineers should be editing copy directly in the app for English, then running `bin/makemessages.sh`, then updating the corresponding Spanish in the PO file.

Reply:
Updated in the dev i18n docs: https://github.com/cal-itp/benefits/pull/1447/files#diff-739a6fd0a2d16786a1fad0499fde882753fee93043bb1f8b1a4b03ba5c75ea06